core/src/main/resources/hudson/logging/LogRecorder/index.jelly

```diff
@@ -29,6 +29,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <l:layout title="Log" permission="${app.SYSTEM_READ}">
   <st:include page="sidepanel.jelly" />
+  <l:breadcrumb title="${%Log}" />
```

Comment:
🐛 Same as for the plugin manager, this does not provide the expected behavior.

It's working for configure/delete but not for index (default page)

<details><summary>Screenshots</summary>

### Before

![Screenshot_2022-11-29_194948_001](https://user-images.githubusercontent.com/2662497/204619517-002a6e1d-2b21-4b11-ba72-0612023efde1.png)


### After

![Screenshot_2022-11-29_194958_001](https://user-images.githubusercontent.com/2662497/204619565-93f38be1-b4b6-486a-b906-7ba5ac89f145.png)



</details>